### DISCUSSION TITLE
Markdown Preview Renderer rendering markdown which is not reflected in Live page

### DISCUSSION THREAD

**Discussion by Dirimsa** [requarks/wiki#6647] [Error / Bug Report] [answered]
Hello,

when I create a new page and try to use some - more specific markdown elements - these are correctly renderer in the markdown preview page on the right side.

However when I save the side these markdown elements are not rendered the same way via markdown.

Examples:

Markdown being correctly rendered in the live preview
![image](https://github.com/requarks/wiki/assets/104820133/37489777-c93a-48fd-b589-8630abb1a351)

After saving the page the following is displayed on the live page:
![image](https://github.com/requarks/wiki/assets/104820133/db60ca13-74c8-4d54-872e-7d6b41f125df)

The correct rendering functionality seems implemented but not in the live page.

Expected Behaviour:

- Rendered markdown is displayed on the live page

Actual Behaviour:

- Raw markdown content is displayed on the live page.

**Comment by NGPixel** [comment, marked as answer]
Did you disable HTML in the markdown renderer?

**Reply by Dirimsa** [reply]
After switching the setting it works perfectly fine. Thanks!

**Reply by orest-tokovenko-deel** [reply]
Hi, i'm having the same issue, could you kindly share where that setting can be found?

**Reply by NGPixel** [reply]
Admin > Rendering > Markdown > Core
